src/common/eeprom.h

```diff
@@ -6,7 +6,7 @@
 #include "variant8.h"
 
 #define EEPROM_ADDRESS  0x0500
-#define EEPROM_VERSION  4
+#define EEPROM_VERSION  6
```

Comment:
why you change from 4 to 6? what does this version number do?

Reply:
This is a mistake. This number should be incremented whenever we add a variable to eeprom. It causes a reset to default values (including the new variable)